src/app/login/login.component.css

```diff
@@ -0,0 +1,48 @@
+.title {
+    text-align: center;
+    color: rgb(238, 105, 28);
+}
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: rgb(161, 219, 219);
+    margin-left: 20px;
+}
+form {
+    border: 3px solid gray;
+    text-align: center;
+}
+input[type=text], input[type=password] {
+    width: 40%;
+    padding: 12px 20px;
+    margin: 8px;
+    display: inline-block;
+    border: 1px solid #ccc;
+    box-sizing: border-box;
+}
+button {
+    background-color: green;
+    color: white;
+    padding: 14px 20px;
```

Comment:
It is always a best practice to add a class to an element then style it rather than adding styles with the tag name as a selector. Prevents style overlapping.